ddl/delete_range.go

```diff
@@ -322,54 +322,38 @@ func insertJobIntoDeleteRangeTable(ctx context.Context, sctx sessionctx.Context,
 		}
 	// ActionAddIndex, ActionAddPrimaryKey needs do it, because it needs to be rolled back when it's canceled.
 	case model.ActionAddIndex, model.ActionAddPrimaryKey:
-		tableID := job.TableID
 		var indexID int64
 		var ifExists bool
 		var partitionIDs []int64
-		var isOldBackfill bool = true
 		if err := job.DecodeArgs(&indexID, &ifExists, &partitionIDs); err != nil {
 			return errors.Trace(err)
 		}
-		tmpID := tablecodec.TempIndexPrefix | indexID
-		if tmpID == indexID {
-			isOldBackfill = false
-			indexID = tablecodec.IndexIDMask & indexID
-		}
+		// Determine the physicalIDs to be added.
+		physicalIDs := []int64{job.TableID}
```

Comment:
what is this line for？ I think line 328 has set partitionIDs already.

Reply:
It is physical ID, not partition ID.